doc/manuals/admin/yum.md

```diff
@@ -0,0 +1,49 @@
+## Using YUM repositories
+
+There are 2 available packages:
+- nightly 
+- release
+ 
+You can manually add config for yum repo or download it from FIWARE public repository.
+
+Use this config for nightly packages:
+```
+[fiware-nightly]
+name=FIWARE nightly repository
+baseurl=http://nexus.lab.fiware.org/repository/el/$releasever/$basearch/nightly
+enabled=1
+gpgcheck=0
+priority=1
+
+```
+or download it from [FIWARE public repository](http://nexus.lab.fiware.org/repository/raw/public/repositories/el/7/x86_64/fiware-nightly.repo)
+```
+curl -o /etc/yum.repos.d/fiware-nightly.repo http://nexus.lab.fiware.org/repository/raw/public/repositories/el/7/x86_64/fiware-nightly.repo
```

Comment:
Note that the general "download command style" in the documentation is wget, instead of curl. Would it be possible to rewrite the command to use wget?

(Same comment in general for the other `curl -o` included in this PR).

Reply:
In addition, for those commands that need root privileges (e.g. creating files in /etc/yum.repos.d/fiware-nightly.repo typically requires that), that should be mentioned.

Reply:
fixed in ce41e406315343b6c61358ad41eda2854b4a1b63